### PR TITLE
=clu #16667 fix Inconsistent message order in ClusterMessageSerializer &...

### PR DESCRIPTION
**Pull request by He-Pin**
porting for master

### CODE REVIEW

File: akka-cluster-metrics/src/main/scala/akka/cluster/metrics/protobuf/MessageSerializer.scala [hunk 1]
```diff
@@ -148,7 +131,7 @@ class MessageSerializer(val system: ExtendedActorSystem) extends Serializer {
         case n: jl.Double  ⇒ Number.newBuilder().setType(NumberType.Double).setValue64(jl.Double.doubleToLongBits(n))
         case n: jl.Long    ⇒ Number.newBuilder().setType(NumberType.Long).setValue64(n)
         case n: jl.Float   ⇒ Number.newBuilder().setType(NumberType.Float).setValue32(jl.Float.floatToIntBits(n))
-        case n: jl.Integer ⇒ Number.newBuilder().setType(NumberType.Integer) setValue32 (n)
+        case n: jl.Integer ⇒ Number.newBuilder().setType(NumberType.Integer) setValue32 n
```
Comment (patriknw) [inline akka-cluster-metrics/src/main/scala/akka/cluster/metrics/protobuf/MessageSerializer.scala]: cosmetic; this should be `.setValue32(n)`

**Comment by patriknw** [comment]
OK TO TEST

**Comment by patriknw** [comment]
LGTM

**Comment by rkuhn** [comment]
LGTM, thanks!

**Comment by patriknw** [comment]
Refs #16667

**Pull request merged**